server/typedb.yml

```diff
@@ -0,0 +1,19 @@
+server:
+    address: 0.0.0.0:1729
+    http_enabled: true
+    http_address: 0.0.0.0:8000
+
+    authentication:
+        token_expiration_seconds: 5000
+
+    encryption:
+        enabled: false
+        # cert: =
+        # cert_key: =
+        # root_ca: =
+
+storage:
+    data: "data"
```

Comment:
should we call this 'directory' too?

Reply:
`storage.directory` doesn't say much to me. You could be storing logs there

Reply:
in the ideal state, we'll have 3 paths:
1. database storage
2. WAL storage
3. Replication storage

i think we should move towards that structure cc @lolski ?

Reply:
Will we have a WAL separate from the raft log?

Reply:
Eventually, but let's push that to later:
```
storage:
  database-directory: <path>
  // future - replication-directory: <path>
```